Tests/DataDefinitionTests.cs

```diff
@@ -108,26 +108,40 @@ public void TestOcellusStationModel()
         }
 
         [TestMethod]
-        public void TestStarSystemExplorationGreenGold()
+        public void TestEmptySystemGreenGold()
+        {
+            StarSystem starSystem = new StarSystem() { systemname = "testSystem" };
+            Assert.IsFalse(starSystem.isgreen, "empty system should not be green");
+            Assert.IsFalse(starSystem.isgold, "empty system should not be gold");
+        }
+
+        [TestMethod]
+        public void TestGreenSystemGreenGold()
         {
             // Set up a body which would trigger the green system condition
             List<MaterialPresence> materials = new List<MaterialPresence>();
-            materials.Add(new MaterialPresence(Material.FromEDName("carbon"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("germanium"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("vanadium"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("cadmium"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("niobium"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("arsenic"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("yttrium"), 0.01M));
-            materials.Add(new MaterialPresence(Material.FromEDName("polonium"), 0.01M));
+            foreach (Material material in Material.jumponiumElements)
+            {
+                materials.Add(new MaterialPresence(material, 0.01M));
+            }
             Body jumponiumBody = new Body()
             {
                 bodyname = "Jumponium Haven",
                 materials = materials
             };
 
+            // Add a body with green materials and re-test
+            StarSystem starSystem = new StarSystem() { systemname = "testSystem" };
+            starSystem.AddBody(jumponiumBody);
+            Assert.IsTrue(starSystem.isgreen, "green system should be green");
+            Assert.IsFalse(starSystem.isgold, "green system should be gold");
```

Comment:
should not be?

Reply:
You got me! Feel free to push a fix.

Assertions w/o comments suck; so do assertions with incorrect comments. Same as all other code.